src/api/request.ts

```diff
@@ -28,18 +33,46 @@ axiosInstance.interceptors.response.use(
     }
   },
 
-  (error: any) => {
-    const { response, status } = error;
-    if (response) {
-      return Promise.reject(response);
+  async (error: any) => {
+    const { data, status } = error?.response || error;
+    const refreshToken = localStorage.getItem('refreshToken');
+    if (refreshToken && data.BusinessCode === '12002') {
+      try {
+        const res: any = await axios({
+          url: `${baseURL}/${authenticationRefreshToken}`,
+          method: 'GET',
+          headers: {
+            Authorization: `Bearer ${refreshToken}`,
+          },
+        });
+        if (res && res.accessToken) {
+          localStorage.setItem('token', res.accessToken);
+          localStorage.setItem('refreshToken', res.refreshToken);
+        }
```

Comment:
If can not refresh token, should redirect to the login page.

Reply:
done